thoth/user_api/api_v1.py

```diff
@@ -239,9 +239,23 @@ def get_analyze_status(analysis_id: str) -> typing.Tuple[typing.Dict[str, typing
     return _get_status("extract-packages", analysis_id, namespace=Configuration.THOTH_MIDDLETIER_NAMESPACE)
 
 
-def post_provenance_python(application_stack: dict, origin: str = None, debug: bool = False, force: bool = False):
+def post_provenance_python(
+    application_stack: dict,
+    debug: bool = False,
+    force: bool = False,
+    # Must be set to set protected fields
+    internal_secret: typing.Optional[str] = None,
+    # Protected fields
+    origin: str = None,
```

Comment:
We want to track origin of the request that is supplied by client (thamos). It might be good not to handled it as a protected.

I think kebechet related inputs are missing here.